src/Request/ConsoleRequest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Bugsnag\BugsnagLaravel\Request;
+
+use Bugsnag\Request\RequestInterface;
+
+class ConsoleRequest implements RequestInterface
+{
+    /**
+     * The unformated console command.
+     *
+     * @var array
+     */
+    protected $command;
+
+    /**
+     * Create a new console request instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->command = $_SERVER['argv'];
```

Comment:
You're right, there's a more laravel way to do it with the Request object.  I'll check that the way to determine cli vs http in base PHP works across laravel and symfony and implement in the base notifier instead.  I'll close this for now.